tests/unit/package_managers/test_gomod.py

```diff
@@ -1566,3 +1577,101 @@ def test_dep_replacements(
         fetch_gomod_source(request)
         expected_calls = [mock.call(request.source_dir / pkg.path, request) for pkg in packages]
         mock_resolve_gomod.assert_has_calls(expected_calls, any_order=True)
+
+
+@pytest.mark.parametrize(
+    "gomod_input_packages, packages_output",
+    (
+        ([], []),
+        (
+            [{"type": "gomod", "path": "."}],
+            [
+                {
+                    "type": "gomod",
+                    "name": "github.com/my-org/my-repo",
+                    "version": "1.0.0",
+                    "path": ".",
+                    "dependencies": [
+                        {
+                            "name": "golang.org/x/net",
+                            "type": "go-package",
```

Comment:
The dependencies of a `gomod` package are all `gomod`. Not that it matters much for this test, but I'd prefer to avoid invalid mocked data

In https://github.com/containerbuildsystem/cachi2/pull/27#discussion_r1032160115, I was refering to the dependencies of the `go-package` package.